WebdevPeriod3/Areas/Identity/Models/ProfileDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using WebdevPeriod3.Areas.Identity.Entities;
+using WebdevPeriod3.Entities;
+
+namespace WebdevPeriod3.Areas.Identity.Models
+{
+    public class ProfileDto
```

Comment:
This still needs to be converted into a proper DTO. Not sure how you you want to do this @thomasio101 since I have only used automapper before.

Reply:
I'll look into this. It might be useful to provide a constructor to more easily create the DTO.

Besides this, we might be over-fetching data here, but given the facts that it would add an unnecessary layer of complexity to fetch specific fields and that we're not sending the raw data to the client with an API, we don't need to worry about limiting the scope of information.

The only other problem I can think of is the potential consequences of showing all the user's products at once. What if there are a LOT of products? We might need to think about pagination.